setup.py

```diff
@@ -44,6 +44,7 @@
             ],
             extra_link_args=[
                 '-lidn2',
+                '-lcpprest',
```

Comment:
make sure to add this dependency inside the github CI/Cd file. Also add it to the readme.md